docs/how-to/troubleshoot_deployment_issues.md

```diff
@@ -0,0 +1,199 @@
+╷
+│ Error: Invalid provider configuration
+│ 
+│ Provider "registry.terraform.io/juju/juju" requires explicit configuration. Add a provider block to the root module and configure the provider's required arguments as described in
+│ the provider documentation.
+│ 
+│ Error: dial tcp 10.152.183.251:17070: i/o timeout
+│ 
+│   with provider["registry.terraform.io/juju/juju"],
+│   on <empty> line 0:
+│   (source code not available)
+│ 
+│ Connection error, please check the controller_addresses property set on the provider
+```
+
+### Recommended Actions
+
+Make sure that the Juju controller is available:
+
+```shell
+$ juju controllers
+Use --refresh option with this command to see the latest information.
+
+Controller           Model      User   Access     Cloud/Region        Models  Nodes  HA  Version
+microk8s-localhost*  private5g  admin  superuser  microk8s/localhost       9      -   -  3.4.5  
+```
+
+If it does not output the controller details, please follow the guide [manage Juju controllers][Manage Juju Controller] to create a Juju controller.
+
+## 5. Charmed Aether SD-Core charms stuck at the Waiting/Blocked status
+
+### Symptoms
+
+After deploying the Charmed Aether SD-Core, check the applications' status:
+
+```shell
+$ juju status
+```
+
+All the charms except the `grana-agent` should be in the Active/Idle state.
+
+If any application hangs in `Waiting` or `Blocked` status, check the table below to see the recommended actions.
+
+### Recommended Actions
+
+If any situation fits to your case in the table below, then perform the recommended actions by utilizing the [Charmed Aether SD-Core Documentation][Charmed Aether SD-Core Documentation].
+
+| Charm Status | How much time passed | Reason                                                                | Recommended Actions                                                   |
+|--------------|----------------------|-----------------------------------------------------------------------|-----------------------------------------------------------------------|
+| Waiting      | For a short time     | Waiting for a relation data, configuration or service to be available | Wait more                                                             |
+| Blocked      |                      | Multus is not installed or enabled                                    | Install and enable Multus                                             |
+| Blocked      |                      | CPU is not compatible                                                 | Use CPU which is Intel 4ᵗʰ generation or newer, or AMD Ryzen or newer |
+| Blocked      |                      | Not enough HugePages available                                        | Set at least two 1G HugePages in the host                             |
+| Blocked      |                      | Waiting for a relation                                                | Find the missing interface and set up the relation                    |
```

Comment:
This seems risky. Especially with Terraform. 
If a relation is missing it's probably a bug in a solution module. 
Also if the user creates a relation manually (`juju integrate`), Terraform will remove it with the first applied change. 
Personally I wouldn't ask the user to fix it on their own.

Reply:
This is indicated as a case that bug report needs to be created.